release-notes/docs/3.9.0.md

```diff
@@ -0,0 +1,163 @@
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [DHIS2 integration](#dhis2-integration)
+- [Supervise target progress](#supervise-target-progress)
+- [RapidPro integration](#rapidpro-integration)
+- [Case management workflows](#case-management-workflows)
+- [Instance monitoring](#instance-monitoring)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+#### Outbound Push only sends each report once
+
+The implementation of Outbound Push has changed as part of [cht-core#6306](https://github.com/medic/cht-core/issues/6306). Now each configured outbound push will be sent only once per doc. Specifically, the first time your `relevant_to` function resolves to `true` is the first and only time a particular record will be send to a configured external service. This is to ensure that pushes are stable and predictable, and additional pushes are not sent unintentionally or unexpectedly. We are working on allowing for multiple pushes, please see [cht-core#6419](https://github.com/medic/cht-core/issues/6419) and contribute your situation to help us determine the best way forward.
+
+#### Cleanup operations may impact app load times
+
+We have implemented a purging function to clean up obsolete feedback and telemetry docs as part of [cht-core#5902](https://github.com/medic/cht-core/issues/5902). This will free up disk space and improve performance over time, however initially may cause slower app load times while the backlog is cleared. The cleanup is capped to 1000 docs per app load to limit the impact on users.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/#dependencies) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## DHIS2 integration
+
+The CHT now supports calculating and exporting data for [DHIS2](https://www.dhis2.org) integration. The Core Framework can now be configured to calculate DHIS2 dataValues, aggregate patient level data, view aggregate data offline, and export the aggregate data formatted for DHIS2. The data is also available via an API so you can access it from an interoperability layer such as OpenHIM.
+
+For more information read [the documentation](https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/).
+
+Issues: [cht-core#5661](https://github.com/medic/cht-core/issues/5661) and [cht-core#6002](https://github.com/medic/cht-core/issues/6002)
+
+## Supervise target progress
+
+Targets can now be configured to be aggregated and synced to allow supervisors to view the progress of their health workers towards their goals. These new "aggregate" targets show an overview of all workers and can be selected to show each worker's progress individually.
+
```

Comment:
Have we got a good screenshot (or screenshots) we can include here?

Reply:
I think this should read:

...These new "aggregate" targets show an overview of all workers progress within their branch and can be selected to show each workers progress individually.

Read the [feature overview](https://docs.communityhealthtoolkit.org/apps/features/targets/#supervisor-view) and the [technical documentation](https://docs.communityhealthtoolkit.org/apps/reference/targets/#targetsjs) to see how to configure this new aggregated target.

^^ feature overview link is not live yet. @abbyad is reviewing PR

Reply:
Updated. Thanks!

Reply:
Marc noticed a mismatch on the previously attached image. Here's a better one

<img width="637" alt="target-detail" src="https://user-images.githubusercontent.com/46463417/84712653-bc41b280-af2e-11ea-949b-5c7931708908.png">

Reply:
Replaced.